content/getting-started/windows.md

```diff
@@ -14,3 +14,35 @@ If you want to use TinyGo to compile your own or sample code, you should be able
 ## Docker Install
 
 The other option is to use the Docker image. This has the benefit of making no changes to your system but has a large download and installation size. For instructions on using the Docker image, please see the page [here](../using-docker).
+
+## Windows Native Install - EXPERIMENTAL
+
+We now have an experimental native install for Windows 10.
+
+VERY IMPORTANT NOTE: You cannot yet create Windows binary programs using TinyGo, only MCU and WASM targets.
+
+- You MUST use Go 1.12.x with the experimental Windows 10 native install of TinyGo.
+
+- Install LLVM 8 from http://releases.llvm.org/download.html#8.0.1
+    Choose the pre-built binary for Windows (64-bit). 
+    During the installation make sure you choose the option to add LLVM to your PATH.
```

Comment:
This is put into a single line by Markdown, which looks a bit weird. Maybe you can add a sublist?

```markdown

- Install LLVM 8 from http://releases.llvm.org/download.html#8.0.1
    * Choose the pre-built binary for Windows (64-bit). 
    * During the installation make sure you choose the option to add LLVM to your PATH.
```

Reply:
Updated.